mlflow/models/evaluation/base.py

```diff
@@ -1667,6 +1667,15 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
     from mlflow.pyfunc import PyFuncModel, _load_model_or_server, _ServedPyFuncModel
     from mlflow.utils import env_manager as _EnvManager
 
+    if evaluator_config is not None:
```

Comment:
Does the col_mapping apply to results.tables["eval_results_table"]? I think it should 

If so, LGTM :)

Reply:
Discussed offline to leave the original name for now :)